backend/src/main/java/com/example/subjecthub/controller/AssessmentServiceController.java

```diff
@@ -0,0 +1,73 @@
+package com.example.subjecthub.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.subjecthub.Application;
+import com.example.subjecthub.api.AssessmentServiceApi;
+import com.example.subjecthub.entity.Assessment;
+import com.example.subjecthub.repository.AssessmentRepository;
+import com.example.subjecthub.utils.FuzzyUtils;
+
+@RestController
+@ParametersAreNonnullByDefault
+@CrossOrigin(origins = "http://localhost:4200")
+public class AssessmentServiceController implements AssessmentServiceApi {
+
+	@Autowired
+	private AssessmentRepository assessmentRepository;
+
+	@Override
+	public List<Assessment> getAssessments(
+			@PathVariable Long universityId,
+			@PathVariable Long subjectId,
+			@RequestParam(required = false) Integer assessmentType,
+			@RequestParam(required = false) Integer weighting,
+			@RequestParam(required = false) Boolean isGroupWork
```

Comment:
Can you change the names to match the fields that we have, so it should just be type, weighting, and groupWork.